Src/BootCamp.Chapter/Checks.cs

```diff
@@ -14,28 +15,73 @@ namespace BootCamp.Chapter
     /// </summary>
     public static class Checks
     {
+        public const int InvalidInt = -1;
+        public const string InvalidString = "-";
+        public const float InvalidFloat = -1;
         public static int PromptInt(string message)
         {
-            // To do: call your implementation. 
-            return 0;
+            Console.WriteLine(message);
+            var input = Console.ReadLine();
+            bool ValidInt;
+            if (String.IsNullOrEmpty(input)) return 0;
+            ValidInt = int.TryParse(input, out var number);
+            if (!ValidInt || number < 0)
+            {
+                Console.Write($"\"{input}\" is not a valid number.");
+                return InvalidInt;
+            }
+            return number;
         }
 
         public static string PromptString(string message)
         {
-            // To do: call your implementation. 
-            return "";
+            Console.WriteLine(message);
+            var input = Console.ReadLine();
+            if (String.IsNullOrEmpty(input))
+            {
+                Console.Write("Name cannot be empty.");
+                return InvalidString;
+            }
+            return input;
         }
 
         public static float PromptFloat(string message)
         {
-            // To do: call your implementation. 
-            return 0;
+            Console.WriteLine(message);
+            var input = Console.ReadLine();
+            if (String.IsNullOrEmpty(input)) return 0;
+            bool ValidFloat = float.TryParse(input, out var bmiInfo);
```

Comment:
Keep variable names consistent- if it's local- camelCase (first letter non-capital).